cypress/e2e/models/administration/jira-connection/jira.ts

```diff
@@ -250,7 +252,7 @@ export class Jira {
     }
 
     public getAllProjects(): Cypress.Chainable<JiraProject[]> {
-        return this.doJiraRequest<JiraProject[]>(`${this.url}/rest/api/3/project`);
+        return this.doJiraRequest<JiraProject[]>(`${this.url}/rest/api/2/project`);
```

Comment:
This number (URL ) remains constant?

Reply:
Yes, it does, as of now both jira cloud and jira datacenter work with the API version 2